browser/src/Input/KeyBindings.ts

```diff
@@ -79,6 +80,7 @@ export const applyDefaultKeyBindings = (oni: Oni.Plugin.Api, config: Configurati
         input.bind("<c-,>", "oni.config.openConfigJs")
 
         if (config.getValue("editor.clipboard.enabled")) {
+            input.bind("<c-x>", "editor.clipboard.cut", isVisualMode)
```

Comment:
`<c-x>` in visual mode decrements an integer value under the cursor

Reply:
Thanks for the heads up, I'll remove this part

Reply:
Well, you're not required to remove it, `<c-v>` is also overriden.

`<c-x>` makes some sense, perhaps `<C-c-x>`?